.github/workflows/run_detekt_and_unit_tests.yml

```diff
@@ -8,10 +8,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.8
+          distribution: 'temurin'
```

Comment:
Just wondering, is it required to specify `distribution: 'temurin'`?

Reply:
@Tuubz Yes it is https://github.com/actions/setup-java/issues/294